code/_onclick/hud/screen_objects.dm

```diff
@@ -29,6 +29,41 @@
 	maptext_height = 480
 	maptext_width = 480
 
+/obj/screen/swap_hand
+	name = "swap hand"
+	name = "swap"
+	icon_state = "swap_1_m"
+	screen_loc = ui_swaphand1
+
+/obj/screen/swap_hand/Click()
+	if(world.time <= usr.next_move)
```

Comment:
These should be handled in the swap_hand itself

Reply:
Now that you mention this, I think it'd be better to remove these altogheter, implementing next_move delay and incapacitate checks into swapping hand will simply make things more frustrating.